webhook/index.ts

```diff
@@ -0,0 +1,117 @@
+import { createWriteStream } from 'fs'
+
+import { v4 as uuidv4 } from 'uuid'
+import express from 'express'
+import execa from 'execa'
+import Bugsnag from '@bugsnag/js'
+import BugsnagPluginExpress from '@bugsnag/plugin-express'
+
+import { WebhookConfig, exists } from './utils'
+
+const { mkdir, readdir, readFile, writeJSON } = require('fs-promise')
+
+const {
+  BUGSNAG_APP_TYPE,
+  BUGSNAG_RELEASE_STAGE,
+  EXPRESS_BUGSNAG_NOTIFIER_KEY,
+  TEMP_DIR
+} = require('../consts.json')
+
+// fs-promise only exports cjs
```

Comment:
Is this comment misplaced?

Reply:
Yes it is! eslint moved it but the comment didn't come with it. Fixed in bd0dd663a5578f8c305459c775f3a7881844d257